runtime/src/accounts_background_service.rs

```diff
@@ -724,6 +735,40 @@ impl AccountsBackgroundService {
     pub fn join(self) -> thread::Result<()> {
         self.t_background.join()
     }
+
+    /// Returns an object to query/manage the status of ABS
+    pub fn status(&self) -> &AbsStatus {
+        &self.status
+    }
+}
+
+/// Query and manage the status of AccountsBackgroundService
+#[derive(Debug, Clone)]
+pub struct AbsStatus {
+    /// Flag to query if ABS is running
```

Comment:
nit: we could optimized this into just one AtomicU8.
0 - running
1 - stopping
2 - stopped.